common/interpreter/wawaka_wasm/WasmExtensions.h

```diff
@@ -43,6 +43,12 @@ bool key_value_get(
     uint8_t** val_buffer_pointer,
     size_t* val_length_pointer);
 
+bool privileged_key_value_get(
```

Comment:
(just a comment for the records)
Exposing the PrivilegedGet to the contract changes the properties of the key-value store, but that's fine.
Originally, the Privileged-* operations where only available to the enclave (not the contract). So the enclave could store data in a separate partition of the KV store, with respect to the contract. As confidentiality for such data is not a requirement, exposing the Get operation does not raise issues. Integrity must (and is) still be preserved, as the contract has read-only access to the enclave's partition in the KV store.